provisioning/ubuntu-provision.sh

```diff
@@ -154,6 +154,9 @@ function install_asdf() {
   echo ". ${asdf_install_dir}/asdf.sh" >> "${profile_script}"
   chown -R "${username}:${groupname}" "${userhome}"
   rm -f "${archive}"
+
+  ## append to the system wide path variable, need to be seconded for docker in packer sources.pkr.hcl
```

Comment:
This comment is the same as the one L211, is it expected?

Reply:
yes